developer/tutorials/articles/03-upgrading-code-to-liferay-7.2/05-fixing-upgrade-problems/02-resolving-a-projects-dependencies.markdown

```diff
@@ -103,7 +103,7 @@ You can use @product@'s
 [App Manager](/docs/7-2/customization/-/knowledge_base/c/finding-artifacts#finding-liferay-portal-app-and-independent-artifacts),
```

Comment:
@jhinkey there were actually two broken links caused by the subheadings issue. The other one is here on line 103. Sorry, I should've mentioned there were two 😓 .

Reply:
Gotcha! No worries. I'll fix em

Reply:
Done. Thanks for pointing it out, Cody!